source/make-an-offer.html.md.erb

```diff
@@ -15,20 +17,26 @@ POST /applications/:id/offer
 POST /applications/11fc0d3b2f/offer
 ```
 
-Request body:
+Request body for an **unconditional** offer:
+
+```json
+<%= unconditional_offer_json %>
+```
+
+Add `conditions` to the request body for a **conditional** offer:
```

Comment:
👌 Typography!

Reply:
Credit to @tijmenb for this! https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training-tech-docs/pull/38#discussion_r322810340